packages/core/database/lib/migrations/index.js

```diff
@@ -26,6 +27,7 @@ const createUmzugProvider = db => {
   const migrationDir = path.join(strapi.dirs.root, 'database/migrations');
 
   fse.ensureDirSync(migrationDir);
+  if (fs.readdirSync(migrationDir).length === 0) fse.writeFileSync(migrationDir + '/.gitkeep', '');
```

Comment:
can you use https://github.com/jprichardson/node-fs-extra/blob/master/docs/ensureFile.md from fs-extra instead

Reply:
yeah sure will swap

Reply:
Actually I can't @alexandrebodin the point of that line is to check if the directory is empty then create the file. fse has no function to check if the directory is empty.

We shouldn't create the `.gitkeep` if there are other files in there so by using `ensureFile` it will always create the field even when it shouldn't.

Reply:
unless you just want me to replace writeFileSync for ensureFile but I don't know if that really changes anything, still can though. 

fs will still be needed to check if dir is empty